main.go

```diff
@@ -77,12 +77,15 @@ func do() (err error) {
 
 	var opts options
 
-	parser := flags.NewParser(&opts, flags.Default)
+	parser := flags.NewParser(&opts, flags.Default & ^flags.PrintErrors)
```

Comment:
I don't feel strongly about it, but I tend to do things this way when I want "whatever the default happens to be, except some specific set", that way if the default changes, this code doesn't need to change.  I also expect it to be easier to review since the reviewer doesn't need to lookup what the default set is and then verify that the newly specified set is equivalent to the default - <undesired-flags>.  But if you feel strongly about it, I can change it.  Let me know.